app/team.go

```diff
@@ -625,11 +625,11 @@ func (a *App) GetAllTeamsPage(offset int, limit int) ([]*model.Team, *model.AppE
 }
 
 func (a *App) GetAllPrivateTeams() ([]*model.Team, *model.AppError) {
-	result := <-a.Srv.Store.Team().GetAllPrivateTeamListing()
```

Comment:
I think we can simplify it even more with just `return a.Srv.Store.Team().GetAllPrivateTeamListing()`